packages/meteor-developer/meteor_developer_client.js

```diff
@@ -40,7 +40,7 @@ var requestCredential = function (options, credentialRequestCompleteCallback) {
     loginUrl: loginUrl,
     credentialRequestCompleteCallback: credentialRequestCompleteCallback,
     credentialToken: credentialToken,
-    popupOptions: {width: 470, height: 420}
+    popupOptions: {width: 470, height: 490}
```

Comment:
Can we not make these configurable within the `ServiceConfiguration`, we have our own Oauth 2 implementation for our customers and are about to switch to dialog based authentication within the next sprint.

When I start work in this I can do a PR for the changes if a team member gives it the thumbs up.